morphic.js

```diff
@@ -7682,19 +7682,21 @@ MenuMorph.prototype.unselectAllItems = function () {
     this.changed();
 };
 
-MenuMorph.prototype.popup = function (world, pos) {
+MenuMorph.prototype.popup = function (world, pos, submenu) {
```

Comment:
Well blame Armando for this habit. But there's a few things:

Boolean rags typically indicate that there should be two functions. A booolean parameter can't indicate anything about what it might control. So now if you later read a call to popup (which should be popUp..) you won't know what the flag is doing.